doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,32 @@
+# InfiniTime Vision
+
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+## InfiniTime
+
+InfiniTime is a community-built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
+
+InfiniTime is not to be used for medical or other health tracking purposes.
+
+## Core Principles
+
+- KISS (Keep It Short and Simple)
+- Reliability
+- Battery efficiency
+- Easy and simple navigation
+- Behaviour should be predictable and easy to understand
+- Prefer solid default experience over customization
+- Personalization is achieved through custom watchfaces.
+  More options may be available through a companion app.
+- Use standard protocols and methods
+
+## Long term vision
+
+The perfect version of InfiniTime would include:
+
+- Capability to upload apps and watchfaces from a phone
+- Only a minimal feature set in the flashed firmware.
```

Comment:
Doesn't this contradict the objective to `prefer solid default experience over customization`?
Imho we should remove non-essential stuff like pong, and move stuff like chimes to standalone apps. But useful features (find my phone, weather, calendar, etc.) should be available out of the box imho.

Reply:
It depends on the interpretation of minimal in this case. We intend to keep basic apps like for example stopwatch, timer and alarm. We'll need to consider which apps should be kept when the time comes, but that'll be a long time from now.

Reply:
I think he idea of 'minimal' is like nothing too crazy-for example no big bowling app game but something like Snake would be acceptable

Reply:
Bring on snake... if this is a serious prospect, I'd love to produce art for it.